src/main/java/org/apache/ibatis/migration/commands/BaseCommand.java

```diff
@@ -152,7 +174,7 @@ private String generatePatternedId(String pattern) {
     }
   }
 
-  private String generateTimestampId() {
+  protected String generateTimestampId() {
```

Comment:
This was changed so that the `NewCommand` subclass could gain access to a timestamp formatting that was already in existence. I was trying to maintain some kind of consistency here since since `timestamp` is a `String` and not a `Date` or something more appropriate.